src/main_menu.cpp

```diff
@@ -4,7 +4,6 @@
 #include "player.h"
 #include "gamemode.h"
 #include "debug.h"
-#include "input.h"
```

Comment:
Since I moved the declaration of `input_context` to the `.h` file, I moved that line there too. Would it be better practice to include it in the `.cpp` file too?